docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/Regression/OrdinaryLeastSquares.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using Microsoft.ML.Data;
+using Microsoft.ML.SamplesUtils;
+
+namespace Microsoft.ML.Samples.Dynamic
+{
+    public sealed class OrdinaryLeastSquares
+    {
+        /// <summary>
+        /// This example require installation of addition nuget package <a href="https://www.nuget.org/packages/Microsoft.ML.HalLearners/">Microsoft.ML.HalLearners</a>
+        /// </summary>
+        public static void Example()
+        {
+            // Downloading a regression dataset from github.com/dotnet/machinelearning
+            // this will create a housing.txt file in the filsystem this code will run
```

Comment:
>this code will run [](start = 68, length = 18)

This sentence does not convey anything?

Reply:
I feel we should have some sort of template kind of thing for all the code and comments because most of the code and comments are shared across many different samples. What do you say? maybe tt template?

---
In reply to: [255788213](https://github.com/dotnet/machinelearning/pull/2497#discussion_r255788213) [](ancestors = 255788213)